.github/workflows/ci.yml

```diff
@@ -358,7 +358,8 @@ jobs:
           ./src/univalue/unitester.exe
 
       - name: Run benchmarks
-        run: ./bin/bench_bitcoin.exe -sanity-check
+        # TODO: Fix the `WalletMigration` benchmark and re-enable it.
+        run: ./bin/bench_bitcoin.exe -sanity-check -filter='^(?!WalletMigration$).+$'
```

Comment:
Yet another alternative would be to comment out `BENCHMARK(WalletMigration, benchmark::PriorityLevel::LOW);` instead (so that when we fix it we won't need to tough ci workflows again)

Reply:
Will it have a broader impact beyond the CI?

Reply:
I've pushed https://github.com/bitcoin/bitcoin/pull/32309 which seems to fix the build (based on https://github.com/l0rinc/bitcoin/pull/9)